docs/controller-skills/military.md

```diff
@@ -69,10 +71,33 @@ These Restricted Areas may be activated in vatSys, and the controllers of these
 Unless otherwise stated in Local Instructions, Restricted Areas activated for control by Military TCUs and Towers, will become **Class C** airspace.
 
 ## Formation Flights
-Reserved.
+A standard military formation consits of a proximity of no more than 1 mile laterally or longitudinally and within 100 feet vertically from the flight leader is maintained by each wingman. As such you treat both aircraft as one and ensure only the leader is on mode C. Futhermore per FIHA, Military AIrcraft are given discrete sqwaks in the 5000's and 6000's.
+
+!!! example
+    **VIPR11**: "Viper Flight Checking in"
+    **ARA**: "Viper flight, sqawk 5000, confirm flight lead is mode C" 
 
 ## Military Authority Assumes Responsibility for Separation of Military Aircraft (MARSA)
-Reserved.
+MARSA is a procedure which authorises pilots of military aircraft to assume
+responsibility for separation between their aircraft and other nominated military aircraft (or military contract
+civil aircraft) within Australian controlled airspace. 
+
+MARSA is to be asssigned and terminated on pilots request and or when standard seperation has been apllied.
+
+!!! example
+    **MAPL01**: "Request MARSA"
+    **DRGN50**: "Request MARSA"
+    **ARA**: "Mantain Block FL270-290, MARSA MAPL01"
+    **ARA**: "Mantain Block FL270-290, MARSA DRGN50"
+    **MAPL01**: "Request MARSA Seperation"
```

Comment:
Small typo of "Separation". I assume this phraseology from this line onwards is for when they're terminating the MARSA? If that's the case, the aircraft will continue to be instructed to do things with "MARSA (callsign)" appended to the end until you have a separation standard. For example:

**MAPL01**: "Request descent to F260 and termination of MARSA"
**ARA**: "MAPL01, Descend to F260, MARSA DRGN50"
(When MAPL01 is maintaining F260):
**ARA**: "MAPL01 and DRGN50, MARSA terminated"